src/basic/MAT_MAT_SHARED.hpp

```diff
@@ -119,6 +119,8 @@ constexpr rajaperf::Index_type TL_SZ = 16;
   if (Row < N && Col < N)                                                      \
     C[Col + N * Row] = Cs[ty][tx];
 
+constexpr int extra_kernel_reps = 5;
```

Comment:
You'll want to make a local variable of this and put it into the DATA_SETUP macro

Reply:
Its probably best to add this as a member variable of KernelBase so the compiler doesn't know how many extra_kernel_reps at compile time.

Reply:
Sure, for now I added as you advised in the first comment, and will also try to do what you advised in this comment. Thanks